### DISCUSSION TITLE
Rate limiting

### DISCUSSION THREAD

**Discussion by darylknight** [verbb/formie#1331] [Ideas]
I'm wondering if it would be possible / planned to add any kind of rate limiting to Formie? I've just had a look through the settings and couldn't see anything. I had an email this morning from a security researcher (badly written spam) which reminded me that coming from Freeform, there's this in the cp:

![image](https://user-images.githubusercontent.com/1250017/222375734-01bfb2e4-4a01-45a4-b694-688077a303a9.png)

Not sure if that's just something that should be done at a higher level, but thought I'd ask.

**Comment by engram-design** [comment]
Nothing specifically planned, but always happy to add this to the roadmap.

**Comment by engram-design** [comment]
Added in [4.0.0-beta.6](https://github.com/verbb/formie/releases/tag/4.0.0-beta.6).
